build.gradle

```diff
@@ -14,10 +14,21 @@ static def calcTags (version) {
 subprojects {
     repositories {
         mavenCentral()
-        jcenter()
         maven {
             url 'https://artifacts.alfresco.com/nexus/content/repositories/public/'
         }
+        // This private repository provides Xenit with Alfresco enterprise artefacts.
```

Comment:
Shouldn't we add Alfresco's enterprise repo instead?

Reply:
Yes, that is also a possibility. I believe we had also credential settings for Alfresco's Nexus enterprise access in the gradle.properties.
I think we should align with @tgeens what approach he prefers.